### ISSUE TITLE
Help us translate CP Editor!

### ISSUE THREAD

**Issue by coder3101** [cpeditor/cpeditor#664]
CP Editor has a thriving community of many users including English, Chinese, Russian, and we are still adding new languages.

However, the developers don't know every language, and the translators don't always have time, so we need your help in translating.

You can read [the contributing guidelines](https://github.com/cpeditor/cpeditor/blob/master/CONTRIBUTING.md#translations) on how to update translations.

We have 3 things you can help us translate
- The markdown files such as [README.md](README.md)
- The in-app GUI elements.
- https://cpeditor.org

Currently, we have translations of the following languages:

- Modern Greek
- Russian
- Simplified Chinese
- Traditional Chinese

If you are familiar with these languages please help us update or improve the translations. New languages are also welcome if anyone is willing to contribute the translations.

**Comment by ouuan** [comment]
I'll write a new CONTRIBUTING.md and describe how to update translations there.

**Comment by CodeCaterpillar666** [comment]
Hello! Is there anything left to translate into Chinese(simplified)? I could translate them!

**Comment by ouuan** [comment]
> Is there anything left to translate into Chinese(simplified)?

No. The Simplified Chinese translations are being actively maintained. However, you can read the current translations and provide some suggestions.

**Comment by ouuan** [comment]
> I would like to propose a translation into Brazilian Portuguese (pt-BR). Please let me know if there are any changes to be made.
> 
> [pt_BR.zip](https://github.com/cpeditor/cpeditor/files/8052360/pt_BR.zip)

Thanks for the translation! Could you open a pull request, or do you want us to open a pull request for you (and you'll appear as the co-author of the commit)?

**Comment by crcpucmg** [comment]
Thanks for you reply.

Feel free to use it as you see fit. I agree that you can open the
pull-request.

I will continue to contribute as much as possible.

My goal was to take an initial step and, for the future, to have the editor
improved and open to other competitions.



Em sex., 11 de fev. de 2022 às 23:06, Yufan You ***@***.***>
escreveu:

**Comment by salmakhaled74** [comment]
I can help you translate to Arabic

**Comment by coder3101** [comment]
@salmakhaled74 #1050 is a Pull request that adds a new translation. 

It would be good if at least 2 people can translate together or someone can review after your translation is done. Feel free ask here if you need further help.

**Comment by TomDevX** [comment]
Hello! I would like to contribute Vietnamese translation. Should I open a new issue before doing the translation?

**Comment by coder3101** [comment]
No need of an issue, you can directly open a PR

**Comment by TomDevX** [comment]
> No need of an issue, you can directly open a PR

Thank you. May I ask that how do I translate on the website?

**Comment by coder3101** [comment]
https://github.com/cpeditor/cpeditor.github.io/blob/hugo/CONTRIBUTING.md#translations

**Comment by TomDevX** [comment]
> https://github.com/cpeditor/cpeditor.github.io/blob/hugo/CONTRIBUTING.md#translations

oh thanks. I thought it just for the app

**Comment by Tejas-Raj01** [comment]
Hi! I would like to contribute a **Hindi (`hi_IN`)** translation for CP Editor.
